CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## [1.61](https://github.com/brave/brave-ios/releases/tag/v1.61)
+
+### Web3
+
+- Implemented new UI for the “Transaction Details” page. ([#8379](https://github.com/brave/brave-ios/issues/8379))
+- Implemented new UI for the “Activity” tab in Brave Wallet. ([#8349](https://github.com/brave/brave-ios/issues/8349))
+- Updated UI and functionality for the "NFT" details page. ([#8064](https://github.com/brave/brave-ios/issues/8064))
+- Updated UI and functionality for “NFT” tab under “Portfolio.” ([#8063](https://github.com/brave/brave-ios/issues/8063))
+- Updated SNS interstitial page copy for new provider. ([#8515](https://github.com/brave/brave-ios/issues/8515))
+- Fixed NFT auto-discover and remove modals to be dismissed when Brave Wallet gets auto-locked. ([#8412](https://github.com/brave/brave-ios/issues/8412))
+
+### General 
+
+- Added share icon to the URL bar. ([#8173](https://github.com/brave/brave-ios/issues/8173))
+- Added separator between inactive tabs. ([#8361](https://github.com/brave/brave-ios/issues/8361))
+- Added  “Redeem Code” button for custom offer codes in the VPN subscription page. ([#8365](https://github.com/brave/brave-ios/issues/8353))
```

Comment:
nit: extra space between `Added` & `Redeem`. The GH issue and the displayed issue number don't match. Link using `8365` but pointing to https://github.com/brave/brave-ios/issues/8353.